cmd/worker/main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/ossf/package-analysis/internal/worker"
```

Comment:
nit: this import should go in the third block after this one.

The order of imports should be:

- stdlibs
- third party packages
- project packages

Reply:
True, `gofmt` doesn't do this naturally so I'll keep a better eye on it